### ISSUE TITLE
ValidatorSite should fallback to HTTP if HTTPS fails

### ISSUE THREAD

**Issue by mellery451** [ripple/rippled#3034]
if a validator list site URL is configured as `https`, but fails for some reason (like cert verification), we should just retry as `http`.

**Comment by MarkusTeufelberger** [comment]
I disagree, there could be very different content and this would also mask errors.

**Comment by mDuo13** [comment]
The validator list is signed by a key that's explicitly configured so at least there's no chance for MITM attacks or whatnot. And since it has a sequence number only one can be valid at a time. Though I suppose something crazy could happen if your list signed two different lists with the same sequence number but that shouldn't happen...
